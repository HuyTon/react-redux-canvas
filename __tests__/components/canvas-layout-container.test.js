import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { Provider } from "react-redux";
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureMockStore from 'redux-mock-store';

// Component to be tested
import CanvasLayoutContainer from '../../src/client/components/containers/canvas-layout-container';

const mockStore = configureMockStore();
const store = mockStore({});

configure({adapter: new Adapter()});
describe('<CanvasLayoutContainer />', () => {
    describe('render()', () => {
        test('renders the component', () => {
            const wrapper = shallow(
                <Provider store={store}>
                    <CanvasLayoutContainer />
                </Provider>);
            const component = wrapper.dive();

            expect(toJson(component)).toMatchSnapshot();
        });
    });
});