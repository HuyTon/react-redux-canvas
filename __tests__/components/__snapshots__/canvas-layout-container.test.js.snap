// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CanvasLayoutContainer /> render() renders the component 1`] = `<Connect(_default) />`;
